roles/ipv6ra/tasks/main.yml

```diff
@@ -11,5 +11,10 @@
             mode=0644
   notify: restart-dnsmasq
 
+- name: prevent dnsmasq from add itself to /etc/resolv.conf via resolvconf
+  lineinfile: dest=/etc/default/dnsmasq regexp="^DNSMASQ_EXCEPT="
+              line="DNSMASQ_EXCEPT=lo"
+  notify: restart-dnsmasq
```

Comment:
This looks to be copied from the neutron-data-network role, tasks/dnsmasq.yml  

In our envs, the `network` hosts, which get neutron-data-network role, is the same as the `controller` hosts. The ipv6ra role is only being ran on the `controller` group. Seem like there is a mismatch of thought here. Should ipv6ra be ran on the network nodes?